chart/templates/_versions.tpl

```diff
@@ -0,0 +1,68 @@
+{{/*
```

Comment:
Please remove this file. See the comment below.

Reply:
Done